src/CorsService.php

```diff
@@ -166,7 +164,7 @@ private function configureAllowedMethods(Response $response, Request $request)
         if ($this->options['allowedMethods'] === true) {
             if ($this->options['supportsCredentials']) {
                 $allowMethods = strtoupper($request->headers->get('Access-Control-Request-Method'));
-                $this->varyHeader($response, 'Access-Control-Request-Method');
+                // Vary is always done
```

Comment:
maybe more specific?
```
// Vary by Access-Control-Request-Method is added to every preflight request.
```

Reply:
Tweaked it a bit by just adding it and detect duplicates, in case the CorsService is used stand-alone or with a custom middleware.